### DISCUSSION TITLE
How to set a default font-family for the button component ?

### DISCUSSION THREAD

**Discussion by kaboume** [vuetifyjs/vuetify#19165] [Q&A] [answered]
Hi,
In my Nuxt3 + Vuetify3 project, I need to set up a specific font-family for the button component.
Is it possible ?
Thanks

**Comment by yuwu9145** [comment, marked as answer]
https://play.vuetifyjs.com/#eNplUMtOwzAQ/JXFF4OUh3j0EtJK9MQRcYAD4WCSNbLk2Ja9iVRV+XfshFRVe9qd2Z3Zx9eRBd+WL84V44CsYjVh77Qg3DUGoB7zDqUYNIXceTuqDn3iAaqV3zbsCAv3sSdTwYoAAh00VgBcWkO5FL3ShwrevCUMBO/KUgZBmJAH9Eo+82wRTilMDUth3mLeQzj3D2bYRkuhDPoTOdM/ZHawH4isgbpc8EkV8aUscatzyq+ujZW6PPtJhKH1yhEEpGERqt5ZT/FwjxImkN72wOM7eWoHiDPjuX34hW3quOWvqLWFT+t1d8Pv0oDFMpqxKWOPxVNxv2Ep2RQP7PsPaWeFbg==
